### ISSUE TITLE
SC 412 -- FAQ disclosure inconsistent state communication

### ISSUE THREAD

**Issue by damiansian** [newjersey/accessibility-pm#12]
## URL 
https://demo.fileyourstatetaxes.org/en/nj/landing-page

## Issue
The screen reader announces Expanded/Collapsed inconsistently. When I open the control, it should announce "expanded". The current implementation does this, but only when I navigate away from it and come back to it. The W3C example announces the expanded and collapsed state each time I open and close the disclosure. The W3C example has all the JS and HTML you'd need to have this occur optimally. 

> [!NOTE]  
> Your current implementation is sufficient for WCAG conformance. This change could take you to that platinum level though. 

https://github.com/user-attachments/assets/4ceac095-3145-4042-90ea-f496afcb077c

## WCAG
[SC 4.1.2: Name, Role, Value (Level A)](https://w3.org/TR/WCAG21#name-role-value)

## User Impact
Screen reader users rely on state information to understand what the interactive control is doing at any given time and in relation to the commands they have made. 

## Suggested fix

Follow the guidance from the Aria Authoring Practices Guide: 
https://codepen.io/dsian108/pen/LEPOBPR

**Comment by tristanvanech** [comment]
might be Honeycrisp fix
